gvsbuild/projects/pango.py

```diff
@@ -24,11 +24,10 @@ def __init__(self):
         Project.__init__(
             self,
             "pango",
-            version="1.50.14",
-            lastversion_even=True,
+            version="1.51.1",
```

Comment:
I do not recall, is 1.51.x a dev release?

Reply:
Hi @nacho, I have no idea actually. There is no 1-50 branch for Pango, there are no clues in the NEWS, and there was similar confusion in the GNOME Hackers room. Fedora, openSUSE, and Arch are using it, so it seems like they think it is stable.

Reply:
https://gitlab.gnome.org/GNOME/pango/-/commit/c732a6431ea5efe86f2f6d8f2b1240459c7b2c74